src/utils/disable-download.ts

```diff
@@ -0,0 +1,11 @@
+import semver from 'semver';
+import { isJest } from './is-jest';
+
+export function disableDownload(version: string) {
+  return (
+    !isJest() &&
```

Comment:
I'm not a big fan of the `isJest` stuff. Can we simply mock the return value for `disableDownload` in the relevant tests where it might mess with snapshots? Simply returning false if it's Jest feels wrong, leading to confusing test behavior like mocking `isJest` to return false.

Reply:
Let me change it later